.github/workflows/create-release.yml

```diff
@@ -0,0 +1,116 @@
+
+          - distro: ubuntu
+            release: focal
+            pkg: deb
+            pkg_name: ubuntu
+
+          - distro: debian
+            release: bullseye
+            pkg: deb
+            pkg_name: debian
+
+    container:
+      image: ${{ matrix.distro }}:${{ matrix.release }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set Version
+        run: |
+           echo "KENTIK_KTRANSLATE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.18.3'
+
+      - name: Get Deps
+        run: |
+          if [ "${{ matrix.distro }}" = "centos" ]
+          then
+            if [ "${{ matrix.release }}" = "7" ]
+            then
+              yum install -y make gcc diffutils libpcap-devel
+            else
+              dnf -y --disablerepo '*' --enablerepo=extras swap centos-linux-repos centos-stream-repos
+              yum install -y make gcc diffutils
+              dnf -y --enablerepo=powertools install libpcap-devel
+            fi
+          else
+            apt-get update
+            apt-get -y install make libpcap-dev ca-certificates build-essential curl
+          fi
+
+      - name: Run Make
+        run: |
+          echo ${{ env.KENTIK_KTRANSLATE_VERSION }}
+          KENTIK_KTRANSLATE_VERSION=${{ env.KENTIK_KTRANSLATE_VERSION }} make
+
+      - name: Run Test
+        run: make test
+
+      - name: Install MM DBs
+        run: |
+          MM_DOWNLOAD_KEY=${{ secrets.MM_DOWNLOAD_KEY }} ./bin/get_mm.sh
+
+      - name: Install SNMP Profiles
+        uses: actions/checkout@main
+        with:
+          repository: kentik/snmp-profiles
+          path: ./config/profiles
+
+      - name: Create package
+        id: package
+        uses: kentik/pkg@master
+        with:
+          name: ktranslate
+          version: ${{ env.KENTIK_KTRANSLATE_VERSION }}
+          arch: x86_64
+          format: ${{ matrix.pkg }}
+          package: ktranslate-package.yml
+
+      - name: Setup Ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.6
+      - run: gem install package_cloud
+
+      - name: Send To Packagecloud
+        env:
+          PACKAGECLOUD_TOKEN: ${{ secrets.PACKAGECLOUD_TOKEN }}
+        run: |
+          REPO="${{ github.event.repository.name }}"
+          package_cloud push kentik/$REPO/${{ matrix.pkg_name }}/${{ matrix.release }} steps.package.outputs.package
+
+      - name: Upload Release Asset
+        id: upload-release-asset
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ github.event.release.upload_url }}
+          asset_path: ${{ steps.package.outputs.package }}
+          asset_name: ${{ steps.package.outputs.package }}--${{ matrix.distro }}-${{ matrix.release }}
+          asset_content_type: application/binary
```

Comment:
One note here: this uploads an asset, but what creates the release? Is that done within the `kentik/pkg` action?

Reply:
Right now, its a manual process to create a release. I click around in the GH webite.